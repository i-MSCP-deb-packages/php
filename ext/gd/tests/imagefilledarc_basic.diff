001+ 68689739f5aadb19efb6a6cf085be676
001- 894f394c7f2e2364642ef27fea6bfc33