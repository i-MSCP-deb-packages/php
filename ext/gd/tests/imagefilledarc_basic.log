
---- EXPECTED OUTPUT
894f394c7f2e2364642ef27fea6bfc33
---- ACTUAL OUTPUT
68689739f5aadb19efb6a6cf085be676
---- FAILED
