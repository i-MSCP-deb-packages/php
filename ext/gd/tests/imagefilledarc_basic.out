68689739f5aadb19efb6a6cf085be676